admin/sql/create_tables.sql

```diff
@@ -42,4 +42,45 @@ CREATE TABLE api_compat.session (
     ts        TIMESTAMP DEFAULT CURRENT_TIMESTAMP
 );
 ALTER TABLE api_compat.session ADD CONSTRAINT session_sid_uniq UNIQUE (sid);
+
+CREATE TABLE statistics.user (
+    user_id                 INTEGER NOT NULL, -- PK and FK to "user".id
+    artists                 JSONB,
```

Comment:
Can you give an example of what these json fields will look like? How often will you read from them or write to them? Will you want to get a specific value from in the field (and therefore require an index)?